src/main/java/dev/task/dndquest/controller/AdventureController.java

```diff
@@ -0,0 +1,23 @@
+package dev.task.dndquest.controller;
+
+import dev.task.dndquest.model.entity.adventure.StoryLine;
+import dev.task.dndquest.service.StoryLineService;
+import java.util.List;
+import lombok.RequiredArgsConstructor;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+@RestController
+@RequestMapping("/api/adventure")
+@RequiredArgsConstructor
+public class AdventureController {
+    private final StoryLineService storyLineService;
+
+    @GetMapping("/all")
+    public String getAvailableAdventures(){
+        String prompt = "generate one json in format \"title\",\"description\" with 3 different dnd quests";
```

Comment:
Controllers should be free from service-level logic. What is acceptable here is some protocol-specific validation, initial parsing and mapping, but not further.
1. Move this logic into service. Having a method called `storyLineService.initStories()` which inside will just call `saveAll(prompt)` will be a good option.
2. Move prompt from variable to constant since it doesn't change. If you will decide to go with option described in this comment - have it in the `StoryLineService` then. Keep in mind Java constant fields conventions.

Reply:
This is temporary, unfinished realization, I'll do it as PlayerController and PlayCharactercontroller. 
Prompt i'm planing to move into separate class (like AiPromptConstants.class) or to the aplication.properties using ConfigurationProperties.